### DISCUSSION TITLE
Automatically expand playlist based on tracks already in playlist (Don't Stop the Music)

### DISCUSSION THREAD

**Discussion by jgillies** [music-assistant/support#2707] [Feature requests and ideas]
One of my favorite features of LMS/Lyrion is _Don't Stop the Music_, which is described [here](https://beta.slimdevices.narkive.com/XN1F1Bx9/slimdevices-new-in-7-9-don-t-stop-the-music). I'd love to see this feature added to Music Assistant.

**Comment by OzGav** [comment]
Already covered here https://github.com/orgs/music-assistant/discussions/813
